stacks/syrah/syrah.yaml

```diff
@@ -1083,6 +1083,13 @@ mpi_blas_python_packages_petsc_gcc_stable:
           gpu:
             nvidia: [petsc +cuda cuda_arch=<cuda_arch>, suite-sparse +cuda]
             none: []
+    - fenics-dolfinx +slepc:
+        dependencies:
+          common:
+            - hdf5 +po +mpi
```

Comment:
This is likely `+ipo` but since there's a typo here I'll wait for your confirmation (and commit).

In any case I'm having issues compiling this, with the error:
```
     42          Reason given by package:
     43              Interpreter: Cannot run the interpreter ".../run/pr/syrah/py3-venv/bin/python3"
```

Reply:
The Python issue was on my side, this now compiles if I put `+ipo`.

I'll wait for your confirmation on whether this is meant to be `+ipo` or for some reason `~ipo`. Otherwise it looks good to me.